Hunting Queries/Syslog/SCXRunAsProviderExecuteShellCommand.yml

```diff
@@ -0,0 +1,41 @@
+id: 0d298a1d-1a08-4f4b-8b28-687bfe0012e8
+name: SCX RunAsProvider ExecuteShellCommand
+description: |
+  'This hunting query uses Auditd security events collected via the Syslog data connector to explore the use of the SCX RunAsProvider Invoke_ExecuteShellCommand to execute any UNIX/Linux command using the /bin/sh shell.
+  SCXcore, started as the Microsoft Operations Manager UNIX/Linux Agent, is now used in a host of products including Microsoft Operations Manager. Microsoft Azure, and Microsoft Operations Management Suite. 
+  '
+severity: High
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - Syslog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+  - Execution
+relevantTechniques:
+  - T1190
+  - T1203
+tags:
+  - SimuLand
+query: |
+  Syslog
+  | parse SyslogMessage with "type=" EventType " audit(" * "): " EventData
```

Comment:
Right after the parse, I would recommend for performance you filter with has on eventdata patterns you match on later, it improves the query time a bunch on larger data sets - | where EventData has "AUOMS_EXECVE" and EventData has '/var/opt/microsoft/scx/tmp'

Reply:
I like that.